pkg/skaffold/render/renderer/renderer.go

```diff
@@ -39,18 +39,32 @@ type Renderer interface {
 }
 
 // New creates a new Renderer object from the latestV2 API schema.
-func New(cfg render.Config, renderCfg latest.RenderConfig, hydrationDir string, labels map[string]string, usingLegacyHelmDeploy bool, command string, configName string) (Renderer, error) {
+func New(cfg render.Config, renderCfg latest.RenderConfig, hydrationDir string, labels map[string]string, usingLegacyHelmDeploy bool, command string, configName string) ([]Renderer, error) {
```

Comment:
how about creating a `RendererOptions` struct, to keep function signature small?
```go
type RendererOptions struct {
    configName string
    hydrationDir string
    labels map[string]string
    usingLegacyHelmDeploy bool
}
```

Reply:
Created a fix it issue to clean this up. 
https://github.com/GoogleContainerTools/skaffold/issues/7704

Thanks
Tejal